### PR TITLE
Fixes #6703: Error about ldconfig during rudder-agent installation on AIX

### PR DESCRIPTION
**Pull request by Kegeruneku**
http://www.rudder-project.org/redmine/issues/6703

### CODE REVIEW

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -745,9 +729,7 @@ rm -f %{_builddir}/file.list.%{name}
 %dir %{ruddervardir}/cfengine-community/inputs
 %dir %{ruddervardir}/tmp
 %dir %{ruddervardir}/tools
-%if 0%{?rhel} != 3
-
-%if "%{use_system_lmdb}" != "true" || "%{use_system_openssl}" != "true"
+%if 0%{?rhel} != 3 && "%{?_os}" != "aix"
 %config(noreplace) /etc/ld.so.conf.d/rudder.conf
 %endif
```
Comment (peckpeck) [inline rudder-agent/SPECS/rudder-agent.spec]: You should remove this one

**Comment by Kegeruneku** [comment]
PR updated

File: rudder-agent/SPECS/rudder-agent.spec [hunk 2]
```diff
@@ -745,15 +729,12 @@ rm -f %{_builddir}/file.list.%{name}
 %dir %{ruddervardir}/cfengine-community/inputs
 %dir %{ruddervardir}/tmp
 %dir %{ruddervardir}/tools
-%if 0%{?rhel} != 3
+%{rudderlogdir}/install
```
Comment (peckpeck) [inline rudder-agent/SPECS/rudder-agent.spec]: What is this doing here ?

**Comment by Kegeruneku** [comment]
PR updated

**Comment by peckpeck** [comment]
OK Good

**Pull request merged**